src/Evolution/DgSubcell/GhostZoneLogicalCoordinates.hpp

```diff
@@ -0,0 +1,36 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+/// \cond
+class DataVector;
+template <size_t Dim>
+class Mesh;
+template <size_t Dim>
+class Direction;
+namespace gsl {
+template <typename T>
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace evolution::dg::subcell::fd {
+
+template <size_t Dim>
```

Comment:
Looks great, would you mind documenting the functions? :)

Reply:
Yupp. Just realized that I forgot docs :joy: